tests/vm_backup_test.go

```diff
@@ -33,10 +37,26 @@ var _ = Describe("[smoke] VM Backup", func() {
 		kvClient, err = util.GetKubeVirtclient()
 		Expect(err).ToNot(HaveOccurred())
 
+		timeout, cancelFunc = context.WithTimeout(context.Background(), 10*time.Minute)
+		t := time.Now().UnixNano()
+		backupName = fmt.Sprintf("test-backup-%d", t)
+		restoreName = fmt.Sprintf("test-restore-%d", t)
+
 		namespace, err = CreateNamespace(client)
 		Expect(err).ToNot(HaveOccurred())
 
-		timeout, cancelFunc = context.WithTimeout(context.Background(), 10*time.Minute)
+		vmSpec := CreateVmWithGuestAgent("test-vm", r.StorageClass)
+		vm, err = CreateVirtualMachineFromDefinition(*kvClient, namespace.Name, vmSpec)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Starting VM")
+		err = StartVirtualMachine(*kvClient, namespace.Name, vm.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = WaitForDataVolumePhase(*kvClient, namespace.Name, cdiv1.Succeeded, vmSpec.Spec.DataVolumeTemplates[0].Name)
```

Comment:
With GC can you be sure the DV is still around? Maybe wait for the VM to have started, the DV will be guaranteed to be completed at that point.

Reply:
well, that means I need to verify the whole test suite! and that is a bigger task. Maybe I can even move the verification to `StartVirtualMachine`

Reply:
but this is a task for another PR.
BTW: is GC enabled by default in all tests?

Reply:
I currently don't think the default is GC, but likely it will be soon.

Reply:
https://github.com/kubevirt/kubevirt-velero-plugin/issues/78